### PR TITLE
[matching] build fix for MSVC 2019

### PR DESCRIPTION
**Pull request by dsmtE**
## Description

Fix https://github.com/alicevision/AliceVision/issues/778

### CODE REVIEW

File: src/aliceVision/matching/kvld/kvld.cpp [hunk 1]
```diff
@@ -431,3 +433,11 @@ float KVLD( const Image< float >& I1,
   return float( matchesFiltered.size() ) / matches.size();
 }
 
+double VLD::get_orientation() const
+{
+    float dy = end_point[1] - begin_point[1];
+    float dx = end_point[0] - begin_point[0];
```
Comment (simogasp) [inline src/aliceVision/matching/kvld/kvld.cpp:439]:
```suggestion
    const float dy = end_point[1] - begin_point[1];
    const float dx = end_point[0] - begin_point[0];
```

File: src/aliceVision/matching/kvld/algorithm.h [hunk 2]
```diff
@@ -108,35 +107,9 @@ inline bool inside( int w, int h, int x,int y, double radius )
...
-inline double angle_difference( const double angle1, const double angle2 )
-{
-	double angle = angle1 - angle2;
-	while( angle <  0 ) angle += 2 * PI_;
-	while( angle >= 2 * PI_ )	angle -= 2 * PI_;
-
-	assert(angle <= 2 * PI_ && angle >= 0 );
-	return std::min( angle, 2 * PI_ - angle );
-}
+double angle_difference(const double angle1, const double angle2);
```
Comment (simogasp) [inline src/aliceVision/matching/kvld/algorithm.h:112]:
```suggestion
double angle_difference(double angle1, double angle2);
```

File: src/aliceVision/matching/kvld/algorithm.h [hunk 3]
```diff
@@ -108,35 +107,9 @@ inline bool inside( int w, int h, int x,int y, double radius )
...
-	if( x < 0 )
-		angle += PI_;
-	while( angle < 0 )
-		angle += 2 * PI_;
-  while( angle >= 2 * PI_ )
-		angle -= 2 * PI_;
-	assert( angle >= 0 && angle < 2 * PI_ );
-	return true;
-}
+bool anglefrom(const float& x, const float& y, float& angle);
```
Comment (simogasp) [inline src/aliceVision/matching/kvld/algorithm.h:110]:
```suggestion
bool anglefrom(float x, float y, float& angle);
```

Comment (dsmtE) [reply src/aliceVision/matching/kvld/algorithm.h:110]: thanks for your suggestions.

**Pull request merged**